packages/ui-components/src/components/ReferenceTooltip.tsx

```diff
@@ -46,39 +46,64 @@ const StyledToolTip = withStyles(theme => ({
   },
 }))(Tooltip);
 
-export interface ReferenceProps {
-  text?: string;
-  name?: string;
-  link?: string;
+interface PlaintextReferenceProps {
+  text: string;
+  name?: never;
+  link?: never;
```

Comment:
Just out of interest, does it just not work if you get rid of the property altogether here?

Reply:
It wouldn’t break anything on a technical level, but then you would be allowed to have the named link properties. This is more to put in guard rails for a dev who might not already know that, semantically, this component only wants one or the other—never both